docs/docs/manual/installing.md

```diff
@@ -7,53 +7,269 @@ sidebar_label: Installing
 
 ## System requirements 
 
+OpenRefine does not require internet access to run its basic functions. Once you download and install it, it runs as a small web server on your own computer, and you access that local web server by using your browser. It only requires an internet connection to import data from the web, reconcile data using a web service, or export data to the web. 
 
+OpenRefine requires three things on your computer in order to function:
 
-*   Java 
-*   Browser
+
+#### Compatible Operating System
+
+OpenRefine is designed to work with Windows, Mac, and Linux operating systems. Our team releases packages for each. 
+
+
+:::tip
```

Comment:
I think they are OK.  Perhaps 1 shade darker. Its not so bad with dark mode on to me. Although I am very used to them now.

Reply:
They're better in dark mode than in light mode, definitely. But I think I will use them more sparingly than they are in this page right now.